example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx

```diff
@@ -171,11 +171,6 @@ const UIRadio = createRadio({
   Label: LabelWrapper,
 });
 
-cssInterop(UIRadio, { className: 'style' });
-cssInterop(UIRadio.Group, { className: 'style' });
-cssInterop(IndicatorWrapper, { className: 'style' });
```

Comment:
Also remove wrappers around it. Don't need those as well.

Reply:
Done, Removed the wrappers.